fs-sim/src/System/FS/Sim/Error.hs

```diff
@@ -476,6 +477,14 @@ instance Arbitrary Errors where
   Simulate Errors monad
 -------------------------------------------------------------------------------}
 
+-- | Alternative to 'mkSimErrorHasFS' that creates 'TVar's internally.
+mkSimErrorHasFS' :: (MonadSTM m, MonadThrow m)
+                => MockFS
```

Comment:
Alignment? I'm not sure how strict we are with it in this repo

Reply:
Good point, I missed that. Done